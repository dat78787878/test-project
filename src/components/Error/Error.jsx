import React from 'react';

const Error = () => {
  return (
    <div className="error">
      Error
      <img
        src="https://www.pngitem.com/pimgs/m/561-5616833_image-not-found-png-not-found-404-png.png"
        alt="not-found"
        width="100px"
        height="100px"
      />
    </div>
  );
};

export default Error;
