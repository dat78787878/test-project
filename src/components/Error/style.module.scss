.error {
  color: #fc0404;
  font-size: 20px;
}
