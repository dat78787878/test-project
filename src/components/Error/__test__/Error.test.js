import React from 'react';
import { render } from '@testing-library/react';
import Error from '../Error';
window.React = React;

describe('Error Component', () => {
  it('render component success', () => {
    const { container, getByText } = render(<Error />);
    expect(container).toMatchSnapshot();
    expect(getByText('Error')).toBeTruthy();
  });
});
