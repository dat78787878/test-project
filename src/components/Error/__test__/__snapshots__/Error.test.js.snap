// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component render component success 1`] = `
<div>
  <div
    class="error"
  >
    Error
    <img
      alt="not-found"
      height="100px"
      src="https://www.pngitem.com/pimgs/m/561-5616833_image-not-found-png-not-found-404-png.png"
      width="100px"
    />
  </div>
</div>
`;
